src/synth/fluid_synth.c

```diff
@@ -554,8 +557,29 @@ new_fluid_synth(fluid_settings_t *settings)
   double gain;
   int i, nbuf;
 
+  
   /* initialize all the conversion tables and other stuff */
-  if (fluid_synth_initialized == 0) {
+  if (fluid_synth_initialized == 0)
+  {
+    char buf[64];
+    if (fluid_settings_str_equal (settings, "synth.volenv", "compliant") == 1)
+    {
+            fluid_conversion_set_atten_power(FLUID_ATTEN_POWER_DEFAULT_COMPLIANT);
+    }
+    else if (fluid_settings_str_equal (settings, "synth.volenv", "emu") == 1)
+    {
+            fluid_conversion_set_atten_power(FLUID_ATTEN_POWER_DEFAULT_EMU);
+    }
+    else
+    {
+        if (fluid_settings_copystr(settings, "synth.volenv", buf, 64) == 1)
```

Comment:
if (fluid_settings_copystr(settings, "synth.volenv", buf, sizeof(buf)) == 1)

Reply:
you're right